crates/volta-migrate/src/lib.rs

```diff
@@ -0,0 +1,134 @@
+#![cfg(feature = "volta-updates")]
+use std::collections::HashSet;
+use std::convert::TryInto;
+use std::fs::{DirEntry, Metadata};
+use std::path::Path;
+
+mod empty;
+mod v0;
+mod v1;
```

Comment:
Style suggestion for here and below: if we follow this up by `use v0::V0` and `use v1::V1`, it makes a *lot* of the rest of the module read a *lot* more cleanly. Thoughts?

Reply:
Good call, I'm not sure why I hadn't done that! I think I might have had competing names at one iteration of this code, but that's not the case any more, so no reason to use qualified names when `use` works!